src/Kriti/Parser/Token.hs

```diff
@@ -154,6 +154,7 @@ data ValueExt
   | Defaulting Span ValueExt ValueExt
   | Range Span (Maybe T.Text) T.Text (V.Vector Accessor) ValueExt
   | EscapeURI Span ValueExt
+  | Function Span T.Text ValueExt
```

Comment:
I think we want the actual haskell function packed into `Function`

Reply:
Marking as resolved